notebooks/MAST/TESS/interm_gi_query/interm_gi_query.ipynb

```diff
@@ -0,0 +1,254 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"imports_ID\"></a>\n",
+    "## Imports\n",
+    "- *Observations* module from astroquery.mast is needed to make the query and download.\n",
+    "- *astropy.io fits* module is needed to view the downloaded fits file. \n",
+    "<p>\n",
+    "For information on how to install astroquery see their [instructions](https://astroquery.readthedocs.io/en/latest/index.html). At the time of writing this requires the latest version on pip."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astroquery.mast import Observations\n",
+    "from astropy.io import fits"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"query_mast\"></a>\n",
+    "## Query the MAST CAOM Database\n",
+    "\n",
+    "We want to retrieve TESS timeseries data (lightcurve and target pixel files) for the Guest Investigator program G011183 from Sector 1 (PI: Stephen Kane). We will need to query the MAST holdings database for the observations.  First we will simply count the number of observations and then we will request a table of those observations. \n",
+    "\n",
+    "All the filter names are listed on the [MAST CAOM Field Description Page](https://mast.stsci.edu/api/v0/_c_a_o_mfields.html). Or, can be found by hovering over the column names after doing a search in the MAST portal. \n",
+    "\n",
+    "The filters we will need to use are, \n",
+    "- **obs_collection** to specify that we want TESS data\n",
+    "- **dataproduct_type** to specify that we want timeseries data\n",
+    "- **sequence_number** to specify that we want sector 1\n",
+    "- **proopsal_id** to specify the GI program number.  \n",
+    "\n",
+    "Remember that more than one GI can propose for the same target so we need wild cards around the name of the program in our query."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "obsCount = Observations.query_criteria_count(obs_collection = \"TESS\",\\\n",
+    "                                       dataproduct_type = [\"timeseries\"],\\\n",
+    "                                           sequence_number = 1,\\\n",
+    "                                          proposal_id = \"*G011183*\")\n",
+    "print(\"Number of Observations: %i\" % obsCount)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "obsTable = Observations.query_criteria(obs_collection = \"TESS\",\\\n",
+    "                                       dataproduct_type = [\"timeseries\"],\\\n",
+    "                                           sequence_number = 1,\\\n",
+    "                                          proposal_id = \"*G011183*\")\n",
+    "obsTable[0:5]['obsid','proposal_id','obs_id']\n"
+   ]
```

Comment:
You don't need the backslashes at the ends of these lines.
Also the query_criteria_count arguments should be lined up under obs_collection.